zerver/openapi/openapi.py

```diff
@@ -291,7 +292,35 @@ def validate_against_openapi_schema(content: Dict[str, Any], path: str,
         # TODO: Remove this after all events have been documented.
         fix_events(content)
     validator = OAS30Validator(schema)
-    validator.validate(content)
+    try:
+        validator.validate(content)
+    except ValidationError as error:
+        to_be_display_schema = {
+            "nullable": False,
+            "oneOf": []
+        }
```

Comment:
Maybe `brief_error_display_schema` and `brief_error_display_validator_value` might be better names, to hint what we're trying to do here?